rkt/image/dockerfetcher.go

```diff
@@ -50,7 +50,10 @@ type dockerFetcher struct {
 // ACI, then stores it in the store and returns the hash.
 func (f *dockerFetcher) GetHash(u *url.URL) (string, error) {
 	ensureLogger(f.Debug)
-	dockerURL := d2acommon.ParseDockerURL(path.Join(u.Host, u.Path))
+	dockerURL, err := d2acommon.ParseDockerURL(path.Join(u.Host, u.Path))
+	if err != nil {
+		return "", fmt.Errorf("invalid docker URL %q. Syntax docker://[REGISTRY_HOST[:REGISTRY_PORT]/]IMAGE_NAME[:TAG]", u)
```

Comment:
Alright, fair enough.

Also, I'm not sure if I like the current error message being two sentences. How about:
```go
fmt.Errorf(`invalid docker URL %q; syntax is "docker://blahblahblah"`, u)
```
Either semicolon or a comma and add a missing "is".